paper-checkbox.html

```diff
@@ -64,9 +64,9 @@
   
     <paper-ripple id="ink" class="circle recenteringTouch" checked?="{{!checked}}"></paper-ripple>
      
-    <div id="checkbox">
-      <div id="checkmark" on-animationend="{{checkboxAnimationEnd}}" on-webkitAnimationEnd="{{checkboxAnimationEnd}}"></div>
-    </div>
+  	<div id="checkbox" class="{{ {checked: checked} | tokenList }}">

```

Comment:
Polymer uses 2 spaces for indentation instead of 4. Can you change the indentation here?